Dockerfile

```
@@ -2,14 +2,14 @@
 ### Base image ###
 ############################################
 
-FROM ruby:3.1.3-slim-buster as base
+FROM ruby:3.1.3-slim-bullseye as base
 LABEL author="IdeaCrew"
 
 RUN apt-get update \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)